pkg/scheduler/factory/factory_test.go

```diff
@@ -453,12 +453,12 @@ func TestInvalidFactoryArgs(t *testing.T) {
 		{
 			name:                           "symmetric weight below range",
 			hardPodAffinitySymmetricWeight: -1,
```

Comment:
We could add another test here when weight is `0`.

Reply:
ok, I will fix it